### PR TITLE
Wrote tests for GetOrganizationByURL, DeleteOrganization, UpdateURL

### PR DESCRIPTION
**Pull request by kinglighthill**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,10 @@ module zuri.chat/zccore
 go 1.16
 
 require (
+	github.com/0xAX/notificator v0.0.0-20210731104411-c42e3d4a43ee // indirect
```
Comment (Aminujb) [inline go.mod:6]: Where is this coming from? I didn't see it usage in the tests you wrote

**Review by Aminujb — COMMENTED** [review]
It'd be a better job to complete all the test cases for a particular function instead of having bits for several functions. This is not going to help us.

File: go.mod (hunk 1 above)
Comment (kinglighthill) [reply go.mod:6]: I don't know

**Comment by kinglighthill** [comment]
> It'd be a better job to complete all the test cases for a particular function instead of having bits for several functions. This is not going to help us.

Those are the complete test for those endpoints. some things there's no means of testing unless the endpoints are modified

**Pull request merged**